plone/app/registry/exportimport/handler.py

```diff
@@ -26,29 +28,20 @@ def evaluateCondition(expression):
 
     ``expression`` is a string of the form "verb arguments".
 
-    Currently the supported verbs are ``installed`` and ``not-installed``.
-    """
-    arguments = expression.split(None)
-    verb = arguments.pop(0)
-
-    if verb not in ('installed', 'not-installed'):
-        raise ValueError("Invalid import condition: %r" % expression)
```

Comment:
@datakurre should there be some bbb code or at least a deprecation warning about not using ``installed`` and ``not-installed``?